issuetracker/pivotaltracker/pivotaltracker_test.go

```diff
@@ -0,0 +1,34 @@
+package pivotaltracker
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_IssueTracker_IssueURLFor(t *testing.T) {
```

Comment:
Could you modify the `Makefile` at the root of the project to also run the unit tests after invoking `make test`?